.github/workflows/automatic-v4-canary-release.yml

```diff
@@ -0,0 +1,20 @@
+name: automatic-v4-canary-release
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  trigger-v4-canary-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger v4 release from main branch
+        run: |
+          curl -L \
+           -X POST \
+           -H "Accept: application/vnd.github+json" \
+           -H "Authorization: Bearer ${{ secrets.GH_CLOUD_SDK_JS_ADMIN_WRITE_TOKEN }}" \
```

Comment:
Not sure if this token has permission to trigger workflows, I don't have admin access in this repo, to manually check.